frontend/packages/console-dynamic-plugin-sdk/src/api/internal-api-kubevirt.ts

```diff
@@ -356,9 +356,7 @@ export const Timestamp = React.lazy(() =>
   })),
 );
 
-export const GenericStatus = React.lazy(() =>
-  import('@console/shared/src/components/status/GenericStatus'),
-);
+export const GenericStatus = React.lazy(() => import('../app/components/status/GenericStatus'));
```

Comment:
as you migrated `GenericStatus` to SDK itself, so do we still need to lazy load it?